README.md

```diff
@@ -275,3 +275,31 @@ The Azure Diagnostic Log Stream ships events from any files in the `D:\home\LogF
                 flushToDiskInterval: TimeSpan.FromSeconds(1))
             .CreateLogger();
 ```
+
+### Pushing properties to the ILogger
```

Comment:
Nit: maybe:

```md
### Pushing properties to `ILogger<T>`
```

?

Reply:
Yes why not, but github didn’t like my formatting, so I converted it to code with ``` around it.